### PR TITLE
LPS-95680 populate SecurityContext

### PR DESCRIPTION
**Pull request by csierra**

### CODE REVIEW

File: modules/apps/portal-remote/portal-remote-jaxrs-security-impl/src/main/java/com/liferay/portal/remote/jaxrs/security/internal/container/LiferaySecurityContextContainerRequestFilter.java [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+				String.valueOf(permissionChecker.getUserId()));
+		}
+
+		@Override
+		public boolean isSecure() {
+			return _portal.isSecure(_httpServletRequest);
+		}
+
+		@Override
+		public boolean isUserInRole(String roleName) {
```
Comment (topolik) [inline modules/apps/portal-remote/portal-remote-jaxrs-security-impl/src/main/java/com/liferay/portal/remote/jaxrs/security/internal/container/LiferaySecurityContextContainerRequestFilter.java]: Carlos, do you see a way to support site roles?

**Comment by topolik** [comment]
@csierra Can you create a test for it? Then I'd prettify it and send to Brian. Thanks.

**Comment by csierra** [comment]
hey @topolik 
I was fixing the rebase problems and creating a test already ;)

**Comment by csierra** [comment]
@topolik Updated PR

**Comment by topolik** [comment]
Hi Carlos, there's a conflict in the PR, looks like somebody already moved directories. Can you please resend?

Thanks.

**Pull request closed without merging**